### PR TITLE
[NEW RELEASE] DAGaml.0.02

### PR DESCRIPTION
**Pull request by JoanThibault**
- **NEW** RBTF, pRBTF, CRBTF, RBTF+max/int
- Upgraded interfaces : OOPS ( cmp, cofactor, id ) and QEOPS ( support )
- **NEW** new module : Support ( interfaced with QEOPS )
- Imported `RBTF_ArgMax_vUInt` from `Snowflake`
- Improved performances of LDD-O-NUCX
- **NEW** Added AQEOPS' library, including : `sat_select_A`, `exists_projA`, `argmaxA`
- **FORMAT** Fixed 'Modele' into 'Model' (spellcheck)

### CODE REVIEW

File: packages/DAGaml/DAGaml.0.02/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+synopsis: "DAGaml : Abstract DAG manipulation in OCaml"
+maintainer: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+authors: "Joan Thibault <joan.thibault@ens-rennes.fr>"
+license: "LGPL-3.0-only with OCaml-LGPL-linking-exception"
+homepage: "https://gitlab.com/boreal-ldd/dagaml"
+bug-reports: "https://gitlab.com/boreal-ldd/dagaml"
+depends: [
+	"ocaml" {>= "4.08.1" & < "4.12"}
```
Comment (kit-ty-kate) [inline packages/DAGaml/DAGaml.0.02/opam:9]: Just curious, what is the reason 4.12 and up aren’t supported?

Comment (kit-ty-kate) [reply packages/DAGaml/DAGaml.0.02/opam:9]: Testing it locally in 4.13 I’m not seeing any issues
```suggestion
	"ocaml" {>= "4.08.1"}
```

Comment (JoanThibault) [reply packages/DAGaml/DAGaml.0.02/opam]: Actually, there is a linking-time error with `GuaCaml.0.03` due to the lack of some hash-related function starting from ocaml 4.12.
So, I added a patch to GuaCaml ( `GuaCaml.0.03.01` ) and fixed DAGaml's dependencies accordingly. =)

**Comment by JoanThibault** [comment]
With regards to the compilation error, it is true that DAGaml only support 64bits architecture (only because of low-level implementation details, its not a global requirement) would it be possible to specify so in the opam file ?

**Comment by mseri** [comment]
Thanks a lot

**Pull request merged**